satellites/adapters/repos/CsvRepository.go

```diff
@@ -0,0 +1,101 @@
+package repos
+
+import (
+	"io"
+
+	"github.com/gocarina/gocsv"
+
+	"github.com/maritza05/golang-bootcamp-2020/satellites/domain"
+)
+
+type CsvSatellite struct {
```

Comment:
Made a separate entity from the one in the domain, they are exactly similar with the difference that this includes tags for the CSV file. I didn't want it to include this information in the domain model so I created another entity specific to the csv and to decouple the model from implementation details.